app/components/footer.js

```diff
@@ -1,19 +1,97 @@
 import React from 'react'
-import { createUseStyles } from 'react-jss'
+import { createUseStyles, ThemeProvider } from 'react-jss'
+import theme from '../theme'
+import ExternalLink from './external-link.svg'
 
 const useStyles = createUseStyles({
```

Comment:
We adopted a convention where we put the styles at the bottom of the file instead of the top. It's uncommon, but we found it was easier to work with that way. We wanted to look at the code first and the styles later.